Componets/Button/button.css

```diff
@@ -0,0 +1,163 @@
+
+.btn-container{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 2.4rem;
+    justify-content: center;
+    align-items: center;
+}
+.btn{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: none;
+    border-radius: 3px;
+    padding: 5px;
```

Comment:
You can use rem instead bcoz padding is we will make it responsive fixed size may need futute changed,It's ok to use px fixed for small

Reply:
i will do it